rdt/transformers/numerical.py

```diff
@@ -58,6 +73,15 @@ def fit(self, data):
         self._dtype = self.dtype or data.dtype
         self.null_transformer = NullTransformer(self.nan, self.null_column, copy=True)
         self.null_transformer.fit(data)
+        if self.min_value == 'auto':
+            self._min_value = min(data)
```

Comment:
I have the following situation and maybe we would like to write a test for it:
```python
data = np.array([1.5, None, 2.5])
# Run
transformer = NumericalTransformer(dtype=np.float, nan='nan', min_value='auto', max_value=None)
transformer.fit(data)
# Error
..........
TypeError: '<' not supported between instances of 'NoneType' and 'float'
```

Reply:
Good catch @pvk-developer 
I suppose we can from the null values from `data` before searching for the min and max